src/Helper/CkoLogger.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Checkoutcom\Helper;
+
+use Checkoutcom\Config\Config;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Client;
+use RuntimeException;
+use Checkoutcom\Helper\Utilities;
+use Psr\Log\LoggerInterface;
+use Monolog\Formatter\JsonFormatter;
+use Checkoutcom\Handler\DatadogHandler;
+
+
+/**
+ * This class is used to log on clouEvent and shopware 6 without
+ * throwing an exception
+ * 
+ */
+class CkoLogger {
+
+    /**
+     * @var LoggerInterface
+     */
+    public static $logger;
+
+    function __construct(LoggerInterface $logger) {
+        self::$logger = $logger;
+    }
+    
+    /**
+     *  log to cloudEvent and shopware 6
+     */
+    public function log($message, $scope, $type, $id = false, $logLevel) {
```

Comment:
`$logLevel` should be an ENUM, you can only provide predefined values (e.g. info, error).